packages/experimental/Badge/src/BadgeColorTokens.ts

```diff
@@ -1,145 +1,493 @@
-        color: globalTokens.color.grey[14],
-        iconColor: globalTokens.color.grey[14],
+        ...getFilledColorProps(
+          {
+            backgroundColor: globalTokens.color.grey[14],
+            backgroundColorDark: globalTokens.color.white,
+            colorDark: globalTokens.color.grey[14],
+          },
+          t,
+        ),
+      },
+      outline: {
+        ...getOutlineColorProps({ color: globalTokens.color.grey[14], colorDark: globalTokens.color.white }, t),
+      },
+      tint: {
+        ...getTintColorProps(
+          {
+            backgroundColor: globalTokens.color.grey[38],
+            color: t.colors.neutralForegroundOnBrand,
+            borderColor: t.colors.transparentBackground,
+            backgroundColorDark: globalTokens.color.grey[68],
+            colorDark: globalTokens.color.grey[16],
+            borderColorDark: globalTokens.color.grey[68],
+          },
+          t,
+        ),
+      },
+      ghost: {
+        ...getGhostColorProps(
+          {
+            color: globalTokens.color.grey[14],
+            colorDark: globalTokens.color.white,
+          },
+          t,
+        ),
       },
     },
-    success: {
+    informative: {
       filled: {
-        backgroundColor: globalTokens.color.green.primary,
-        color: t.colors.neutralForegroundOnBrand,
-        iconColor: t.colors.neutralForegroundOnBrand,
+        ...getFilledColorProps(
+          {
+            backgroundColor: globalTokens.color.grey[92],
+            color: globalTokens.color.grey[38],
+            backgroundColorDark: t.colors.black,
+            colorDark: globalTokens.color.grey[68],
+            hcBackground: t.colors.neutralBackground3,
+            hcColor: t.colors.brandForeground1,
+            hcBorderColor: t.colors.brandForeground1,
+          },
+          t,
+        ),
       },
       outline: {
-        color: globalTokens.color.green.primary,
-        iconColor: globalTokens.color.green.primary,
-        borderColor: globalTokens.color.green.primary,
+        ...getOutlineColorProps(
+          { color: globalTokens.color.grey[92], colorDark: globalTokens.color.grey[68], borderColorDark: globalTokens.color.grey[32] },
+          t,
+        ),
       },
       tint: {
-        backgroundColor: globalTokens.color.green.tint60,
-        color: globalTokens.color.green.primary,
-        iconColor: globalTokens.color.green.primary,
-        borderColor: '#A0D8A0',
+        ...getTintColorProps(
+          {
+            backgroundColor: globalTokens.color.grey[94],
+            color: globalTokens.color.grey[38],
+            borderColor: globalTokens.color.grey[92],
+            backgroundColorDark: globalTokens.color.grey[8],
+            colorDark: globalTokens.color.grey[68],
+            borderColorDark: globalTokens.color.grey[32],
+          },
+          t,
+        ),
+      },
+      ghost: {
+        ...getGhostColorProps(
+          {
+            color: globalTokens.color.grey[92],
```

Comment:
> color: globalTokens.color.grey[92], [](http://example.com/codeflow?start=11&length=36)

This doesn't seem right, this would be close to white on top of a light background, is 92 correct?

Reply:
Yes, it's according to the design:
![image](https://user-images.githubusercontent.com/11574680/184135520-40bdb09f-a3d4-470c-a3ab-cf86a9d6c464.png)